### ISSUE TITLE
JSON error while convert currencies

### ISSUE THREAD

**Issue by difujia** [biati-digital/alfred-calculate-anything#76]
If you have a problem with the workflow please answer the following questions.

**Are you using the latest version of the workflow?**
Yes, v3.5.0

**Describe the bug**

Cannot convert currencies

**To Reproduce**
Type any currency conversion syntax like: 100 jpy to usd
Observed: no result shown

**Expected behavior**
Conversion result should be shown

**Debug Output**
![image](https://user-images.githubusercontent.com/2255369/180162021-e7f53108-c45d-4119-9b60-8f56946f402a.png)

JSON error: JSON text did not start with array or object and option to allow fragments not set. around line 1, column 0. in JSON:

Please enable debug (see image) open Alfred and type the conversion that it's not working, you will see that the debug window is populated with a lot of text, please paste that text in here.

**System information:**
 - OS: macOS 12.4
 - Alfred Version 5
- PHP Version 8.1.8

**Additional context**

**Comment by biati-digital** [comment]
Hi, did you configure your API Key?

**Comment by difujia** [comment]
> Hi, did you configure your API Key?

No I don't

**Comment by nchieng** [comment]
I've obtained a Fixer API key from API Layer, and configured the workflow with it. (v 3.5.0)
The problem still persists.

```
[12:35:38.816] ERROR: Calculate Anything[Script Filter] JSON error: JSON text did not start with array or object and option to allow fragments not set. around line 1, column 0. in JSON:
Warning: Undefined array key "message" in /Users/<uname>/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.<>/workflow/tools/currency.php on line 467

Warning: Trying to access array offset on value of type float in /Users/<uname>/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.<>/workflow/tools/currency.php on line 287

Warning: Trying to access array offset on value of type float in /Users/<uname>/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.<>/workflow/tools/currency.php on line 289
```

**Comment by biati-digital** [comment]
There's a new version [4.0.0](https://github.com/biati-digital/alfred-calculate-anything/releases/tag/4.0.0). Please read the release to see what's new.

Please note, the only way for currency to not work it's because it's not able to grab the rates from fixer or exchangeratehost, are you using a software to block outgoing connections?

If you are using Alfred 5 you can migrate you configuration to the new workflow configuration window. The workflow should still be able to read the old config but try it anyway.

**Comment by biati-digital** [comment]
A new update has been released [4.0.1]( https://github.com/biati-digital/alfred-calculate-anything/releases/tag/4.0.1) this one should solve this situation, seems to be the same as #103
